taxonomies/nutrient_levels.txt

```diff
@@ -34,7 +34,7 @@ cu:Fat in low quantity
 cv:Fat in low quantity
 cy:Braster in low quantity
 da:Fedt i lav mængde
-de:Fett in geringen Mengen
```

Comment:
I think you did a `git add .` here and added these files without notice, make sure to add only the file you want to with `git add {myfile}`